lib/measured/measurable.rb

```diff
@@ -11,8 +11,10 @@ def initialize(value, unit)
     @value = case value
     when Float
       BigDecimal(value, Float::DIG + 1)
-    when BigDecimal
+    when BigDecimal, Rational
       value
+    when Integer
+      Rational(value)
```

Comment:
`Rational` will coerce to `BigDecimal` when in an operation involving a `BigDecimal`. The idea here is to stick with `Rational` if it would be desirable, but since parsing is potentially much slower, we use `BigDecimal` for the `else` clause.